builtin_op_importers.cpp

```diff
@@ -981,6 +981,25 @@ DEFINE_BUILTIN_OP_IMPORTER(Identity) {
   RETURN_IDENTITY(inputs.at(0));
 }
 
+DEFINE_BUILTIN_OP_IMPORTER(ImageScaler) {
+    ASSERT(inputs.at(0).is_tensor(), ErrorCode::kUNSUPPORTED_NODE);
+    OnnxAttrs attrs{node};
+    // Shift the input by a per-channel bias value.
+    std::vector<float> biases = attrs.get<std::vector<float>>("bias");
+    nvinfer1::Dims dims{1, static_cast<int>(biases.size())};
+    ShapedWeights shiftWeights = ctx->createTempWeights(::ONNX_NAMESPACE::TensorProto_DataType_FLOAT, dims);
+    std::memcpy(shiftWeights.values, biases.data(), biases.size());
```

Comment:
biases.size() is in elements, not bytes!
I suggest casting the values pointer and using std::copy instead.

Reply:
Good catch, updated.